src/ServicePulse.Host/app/js/views/monitored-endpoints/monitored-endpoints.module.js

```diff
@@ -17,10 +17,9 @@
                         var width = 130;
                         var margin = 2;
                         var average = d3.mean(scope.data);
-                        var max = d3.max(scope.data);
-                        var min = d3.min(scope.data) !== d3.max(scope.data) ? d3.min(scope.data) : 0;
+                        var max = Math.max(average * 1.5, d3.max(scope.data));
```

Comment:
It is rather minor but i wonder if as a max we shouldn't just get (d3.max(scope.data) * 1.1). So we take the maximum value and add 10% to it. Which means that highest point in the diagram will still have a bit of space to the top of the chart. But we can decide on that at later stage